device/build.gradle

```diff
@@ -8,8 +42,8 @@ android {
         versionCode 3
         versionName "0.3"
 
-        minSdkVersion 20
```

Comment:
I think it is better if it not derived from rootProject, because in this case it possible to build the wear and device to different platforms